runelite-client/src/main/java/net/runelite/client/plugins/timetracking/farming/PatchImplementation.java

```diff
@@ -2603,6 +2605,137 @@ PatchState forVarbitValue(int value)
 				}
 				return null;
 			}
+		},
+	COMPOST(Tab.SPECIAL, "Compost Bin")
+		{
+			@Override
+			PatchState forVarbitValue(int value)
+			{
+				if (value == 0)
+				{
+					// Compost bin[Examine] 7808
```

Comment:
fwiw Examine isn't a real menu op and shouldn't be here, but this is just debug from the code generator script so you don't need it anyway

Reply:
I made these not realizing the stuff was code gen'd so I tried to copy the style of the existing code